src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/scaleset/AzureScaleSetProvider.java

```diff
@@ -0,0 +1,327 @@
+
+                                                                                                                                               .withRegion(region)
+                                                                                                                                               .withExistingResourceGroup(resourceGroup)
+                                                                                                                                               .withSku(selectScaleSetSku(instance.getHardware()
+                                                                                                                                                                                  .getType()))
+                                                                                                                                               .withExistingPrimaryNetworkSubnet(network,
+                                                                                                                                                                                 azureSubnetName)
+                                                                                                                                               .withExistingPrimaryInternetFacingLoadBalancer(lb)
+                                                                                                                                               .withPrimaryInternetFacingLoadBalancerBackends(azureBackendPoolName)
+                                                                                                                                               .withPrimaryInternetFacingLoadBalancerInboundNatPools(azureNATPoolSSHName,
+                                                                                                                                                                                                     azureNATPoolPNPName,
+                                                                                                                                                                                                     azureNATPoolPNPSName)
+                                                                                                                                               .withoutPrimaryInternalLoadBalancer()
+                                                                                                                                               .withPopularLinuxImage(selectScaleSetLinuxImage(imageNameOrId))
+                                                                                                                                               .withRootUsername(vmAdminUsername);
+        VirtualMachineScaleSet.DefinitionStages.WithLinuxCreateManagedOrUnmanaged virtualMachineScaleSet_stage2;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00117)